mail/urlscan/Portfile

```diff
@@ -0,0 +1,27 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           python 1.0
+
+github.setup        firecat53 urlscan 0.9.5
+categories          mail python
+platforms           darwin
+license             GPL-2
+maintainers         {@ryanakca debian.org:rak}
+description         extract and browse the URLs in an email
+long_description    ${name} lets you easily browse URLs contained in an email\
+                    and launch a web browser to view them. It is a replacement\
+                    for the "urlview" program and integrates with the mutt MUA.
+
+github.tarball_from archive
+
+python.default_version     38
+
+checksums           rmd160  439e24ccb82619affe45c22bbca2272d466b875a \
+                    sha256  db76f61966431a8938adc998aaa2b30be54f9b1ff2a4c0064e9915df2fb6b996 \
+                    size    32010
+
+depends_build-append port:py${python.version}-setuptools
+
+depends_lib         port:py${python.version}-urwid
```

Comment:
please change this to `depends_lib-append`, right now you're overwriting the dependency on `python38` that is set by the python PG